src/lib/dom-api.html

```diff
@@ -593,6 +593,61 @@
 
     };
 
+    Object.defineProperties(DomApi.prototype, {
+      activeElement: {
+        configurable: true,
+        get: function() {
+          var ownerDocument = this.node.ownerDocument;
+
+          // Shadow DOM
+          if (Polymer.Settings.useShadow) {
+            if (ownerDocument || !window.ShadowDOMPolyfill) {
+              return this.node.activeElement;
+            } else {
+              return window.ShadowDOMPolyfill.wrapIfNeeded(this.node).activeElement;
+            }
+          }
+
+          // Shady DOM
+
+          var trueDocument = ownerDocument || this.node;
+
+          // Not a document or shady root.
+          if (ownerDocument && !this.node._isShadyRoot) {
+            return this.node.activeElement;
```

Comment:
This only makes sense if this.node == document doesn't it? If so, let's make that explicit `if (this.node === document)`...